access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+|`Describe`
+|===
+--
+. Use the *Permissions* page to set permissions for a new account, add permissions to an existing account, or delete account permissions in this instance.
++
+To set permissions for a new account in this instance, follow these steps:
+
+.. Click *Manage access*.
+.. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
+.. Click *Next*.
+.. Under *Review existing permissions*, verify the current permission settings for this instance. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Permissions* page.
++
+If you previously selected a specific account, you can delete only permission entries that apply to individual accounts. If you previously selected *All accounts*, you can delete only permission entries that apply to all accounts.
+
+.. Under *Assign Permissions*, set the permissions for the specified account or all accounts for a resource type, such as a topic, and click *Add* to continue adding permissions for other resources as needed.
++
+--
+For example, when you create a new service account, you can add the following permissions in the Kafka instance that you want the account to access. In this example, the permissions enable applications associated with the service account to produce and consume messages in any topic in the instance, using any consumer group and any producer.
+
+.Example ACL permissions for a new service account
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operation
+
+|`Topic`
+|`Is` = `*`
+|`Allow`
+|`All`
+
+|`Consumer group`
+|`Is` = `*`
+|`Allow`
+|`Read`
+
+|`Transactional ID`
+|`Is` = `*`
+|`Allow`
+|`All`
+|===
+--
+.. Click *Save* to finish.
+
+To add permissions to an existing account in this instance, follow these steps:
+
+.. For the account that you want to add permissions to, select the options icon (three vertical dots) for that entry and click *Manage*.
+.. Under *Review existing permissions*, verify the current permission settings for this instance. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Permissions* page.
++
+If you selected a permission entry that applies to a specific account, you can delete only permission entries that apply to individual accounts. If you selected a permission entry that applies to all accounts, you can delete only permission entries that apply to all accounts.
+
+.. Under *Assign Permissions*, set the additional permissions for that account or all accounts for a resource type, such as a topic, and click *Add* to continue adding permissions for other resources as needed.
+.. Click *Save* to finish.
+
+To delete existing account permissions in this instance, use the following options:
+
```

Comment:
Again, this feels like we have a lot of procedures in a single module. (Or maybe I'm looking at this incorrectly?)

Reply:
Ditto